docs/design-docs/overview.md

```diff
@@ -0,0 +1,144 @@
+---
+id: overview
+---
+
+# Superbench Design
+
+## Goals
+
+Extend current tool to fit distribution test needs in cluster with 100 ~ 1000 nodes, make it modulable, easy to use and easy to scale up.
+Provide more simple and convenient way to:
+1.	Install and deploy the tool in a raw cluster, including bare metal, on-premises, and cloud environments.
+2.	Configure super benchmark configurations, by specifying config file or using command line arguments.
+3.	Launch super benchmark locally, on selected nodes, or on all nodes in the cluster. Use single device, selected devices, or all devices, including GPU and InfiniBand.
+4.	Be able to run with different distribution mode, e.g., mpi, torch distributed, ps/worker or client/server communication, etc.
+5.	Collect log during running and save results after running, merge results from all nodes to single one.
```

Comment:
Collect log during running, save results after running, and merge all nodes' results to one summary report.